diff --git a/isotools-jessie/debian-cd/CONF.sh b/isotools-jessie/debian-cd/CONF.sh
index 11c613d..ffea236 100644
--- a/isotools-jessie/debian-cd/CONF.sh
+++ b/isotools-jessie/debian-cd/CONF.sh
@@ -59,7 +59,7 @@ export BASEDIR=`pwd`
 # export CDNAME=debian
 
 # Building wheezy cd set ...
-export CODENAME=wheezy
+export CODENAME=jessie
 
 # By default use Debian installer packages from $CODENAME
 if [ -z "$DI_CODENAME" ]; then
--- a/isotools-jessie/debian-cd/CONF.sh
+++ b/isotools-jessie/debian-cd/CONF.sh
@@ -76,7 +76,7 @@ fi
 #export DI_WWW_HOME=default
 
 # Version number, "2.2 r0", "2.2 r1" etc.
-export DEBVERSION="7.0"
+export DEBVERSION="8.0"
 
 # Official or non-official set.
 # NOTE: THE "OFFICIAL" DESIGNATION IS ONLY ALLOWED FOR IMAGES AVAILABLE
@@ -115,7 +115,7 @@ export MIRROR=/srv/mirror/debian
 export TDIR=/srv/mirror/tmp
 
 # Path where the images will be written
-export OUT=/srv/mirror/debian-cd-test
+export OUT=/tmp/untangle-images
 
 # Where we keep the temporary apt stuff.
 # This cannot reside on an NFS mount.
diff --git a/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86 b/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86
index c6d397c..b523270 100644
--- a/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86
+++ b/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86
@@ -429,6 +429,11 @@ if [ -n "$KERNEL_PARAMS" ]; then
     done
 fi
 
+rm -f boot$N/isolinux/{*txt,*cat,ad*,exit*,prompt*,rq*,spk*}
+cp -f ../../../cd-root/isolinux/* boot$N/isolinux/
+rm -fr CD$N/README.*
+cp -r ../../../cd-root/* CD$N/
+
 set_default_desktop
 rm -rf cdrom
 
